packages/xrpl/src/models/methods/ledgerEntry.ts

```diff
@@ -204,5 +207,9 @@ export interface LedgerEntryResponse<T = LedgerEntry> extends BaseResponse {
     /** The binary representation of the ledger object, as hexadecimal. */
     node_binary?: string
     validated?: boolean
+    /**
+     * If the object was deleted in this ledger, this field contains the sequence number of the transaction that deleted it
```

Comment:
should change the comment to match what is written in the design/spec. should mention it's clio-only